<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 15px; /* 모서리 둥글기 설정 */
    background-color: rgba(113, 199, 236, 0.5); /* 투명도 50%의 녹색 */
    color: white; /* 글자색 */
    font: bold 14px; /* 폰트 스타일 */
    padding: 6px; /* 패딩 */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>arrow.png</normaloff>arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>841</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-image: url('C:/Users/user/WP/AIconvergence/ultralytics/map2.png');
    background-repeat: no-repeat;
    background-position: center;
}</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
